src/FriendsPage.tsx

```diff
@@ -102,24 +103,27 @@ const FriendsPage: React.FC = () => {
         "An error occured."
       ) : (
         <>
-          <AutoTable
-            data={friendRequests}
-            title={<h3>Friend Requests</h3>}
-            hideOnEmpty
-          >
-            <TableColumn col={"display_name"}>Username</TableColumn>
-            <FriendRequestButtons
-              onAccept={AcceptFriendship}
-              onReject={RejectFriendship}
-            />
-          </AutoTable>
-          <AutoTable
-            data={currentFriends}
-            title={<h3>Current Friends</h3>}
-            hideOnEmpty
-          >
-            <TableColumn col={"display_name"}>Username</TableColumn>
-          </AutoTable>
+          <FriendSearchBar />
+          <div style={{ zIndex: 0, position: "sticky" }}>
```

Comment:
Should this be in a styled component? Also how does this affect scrolling on a phone?

Reply:
Ah, yeah. I originally put that in because I was having z-index errors (the dropdown was behind the table), but those seem to have been resolved. It's been removed.